client/src/Movies/MovieList.js

```diff
@@ -35,20 +37,8 @@ function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
     <div className="movie-card">
-      <h2>{title}</h2>
-      <div className="movie-director">
-        Director: <em>{director}</em>
-      </div>
-      <div className="movie-metascore">
-        Metascore: <strong>{metascore}</strong>
-      </div>
-      <h3>Actors</h3>
-
-      {stars.map(star => (
-        <div key={star} className="movie-star">
-          {star}
-        </div>
-      ))}
+    <Link  to={`/movies/${movie.id}`}>
+    <MovieCard movie={movie} />
+      </Link> 
```

Comment:
👍 note: please indent the component based on its nested position for readability. JS doesn't if you do this or not, but some other languages like Python do.